### PR TITLE
Automated release of 31.0.0 upstream release 31.0 1726263540

### PR DESCRIPTION
**Pull request by yliaog**
#### What type of PR is this?
/kind bug

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# v31.0.0
+
+Kubernetes API Version: v1.31.1
+
+### API Change
+- The resource/v1alpha3.ResourceSliceList filed which should have been named "metadata" but was instead named "listMeta" is now properly "metadata". ([kubernetes/kubernetes#126761](https://github.com/kubernetes/kubernetes/pull/126761), [@thockin](https://github.com/thockin)) [SIG API Machinery]
```
Comment (roycaihw) [inline CHANGELOG.md:6]: I didn't see any generated client change regarding this (https://github.com/kubernetes/kubernetes/pull/126761/files). Do you know why is that?

Comment (yliaog) [reply CHANGELOG.md:6]: we are release 1.31.0, https://github.com/kubernetes/kubernetes/releases/tag/v1.31.0

the release script is pulling the openapi from 1.31.1. i guess we can manually edit the CHANGELOG to reflect that we are actually release 1.31.0, WDYT?

Comment (roycaihw) [reply CHANGELOG.md:6]: > i guess we can manually edit the CHANGELOG to reflect that we are actually release 1.31.0, WDYT?

Yes I think that works. Let's do that. Could you update this PR as well as https://github.com/kubernetes-client/python/pull/2282 and let me know when it's ready?

Comment (yliaog) [reply CHANGELOG.md]: done

**Review by roycaihw — APPROVED** [review]
/lgtm
/approve

**Pull request merged**